PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -521,6 +531,11 @@ BOOL WindowsDlg::onInitDialog()
 	lvColumn.cx = 50;
 	SendMessage(_hList, LVM_INSERTCOLUMN, 2, LPARAM(&lvColumn));
 
+	columnText = TEXT("\u21F5 ") + pNativeSpeaker->getAttrNameStr(TEXT("Size"), WD_ROOTNODE, WD_CLMNNAME);

```

Comment:
@eljefe7000

FWIW, this change won't properly translate "size". At present, the column will be translated to "name". See 13801e3540 for guidance.

Also, I bet the "size" translation can be copied from Styler Dialog's `id="25032"` in every language's translation file. Examples: [English](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/13801e35405841cfb8ff0af65023e341ba3dca34/PowerEditor/installer/nativeLang/english.xml#L496) & [French](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/13801e35405841cfb8ff0af65023e341ba3dca34/PowerEditor/installer/nativeLang/french.xml#L538)